plugins/com.google.cloud.tools.eclipse.googleapis/OSGI-INF/com.google.cloud.tools.eclipse.googleapis.internal.GoogleApiFactory.xml

```diff
@@ -3,6 +3,6 @@
    <service>
       <provide interface="com.google.cloud.tools.eclipse.googleapis.IGoogleApiFactory"/>
    </service>
-   <reference bind="setProxyService" cardinality="0..1" interface="org.eclipse.core.net.proxy.IProxyService" name="ProxyService" policy="dynamic" unbind="unsetProxyService"/>
+   <reference bind="setProxyService" cardinality="0..1" interface="IProxyService" name="ProxyService" policy="dynamic" unbind="unsetProxyService"/>
```

Comment:
Should we revert this? My Eclipse restore this to `org.eclipse.core.net.proxy.IProxyService` automatically.

Reply:
Should be reverted: the interface is a fully-qualified type name.

Reply:
will revert, though why this happened I don't know. I didn't touch this file. Eclipse just decided it needed to be other than it was. :-(